src/Notes/ReviewShippingSettings.php

```diff
@@ -27,18 +27,23 @@ class ReviewShippingSettings {
 	public static function get_note() {
 		$note = new Note();
 
-		$note->set_name( self::NOTE_NAME );
-		$note->set_source( 'woocommerce-admin' );
-		$note->set_title( __( 'Review your shipping settings', 'woocommerce-admin' ) );
-		$note->set_content(
+		$note_lines = array(
```

Comment:
Would `$content_lines` describe this better?

Reply:
Good point! Yup, I think `$content_lines` makes more sense. I pushed a new commit.